python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py

```diff
@@ -247,15 +247,19 @@ def optimize(self, model: Model,
                 def func_test(model, sample):
                     model(sample)
                 try:
-                    if method == "original" and thread_num is not None:
+                    if method in ("original", "static_int8") and thread_num is not None:
                         _flag = True  # represent whether subprocess works
                         # for original keras model, as tf.config.threading can't set thread
                         # during running, so here we use subprocess to calculate throughput
                         params = {"iterrun": latency_sample_num,
                                   "func": func_test,
-                                  "model": acce_model,
-                                  "input_sample": input_sample}
+                                  "model": model,  # save original model
+                                  "input_sample": input_sample,
+                                  "method": method}
                         with tempfile.TemporaryDirectory() as temp_dir:
+                            if method != "original":
+                                # save accelerated model
+                                InferenceOptimizer.save(acce_model, temp_dir)
```

Comment:
besides, I wonder if `cloudpickle` can dump inc model directly.

Reply:
I have tried this, `cloudpickle` can't dump inc model.

Reply:
all right